### ISSUE TITLE
[Feature] Does mmdeploy support MMYOLO related model?

### ISSUE THREAD

**Issue by kelvinwang139** [open-mmlab/mmdeploy#1882]
### Motivation

I notice that the MMYOLO is independent but mmdeploy seems no official announce to support it. Does mmdeploy support MMYOLO related model?

### Related resources

https://github.com/open-mmlab/mmyolo

**Comment by lvhan028** [comment]
Yes. It does. Please refer to https://github.com/open-mmlab/mmyolo/blob/main/docs/en/recommended_topics/deploy/mmdeploy_guide.md

**Comment by caj-github** [comment]
![image](https://user-images.githubusercontent.com/55951697/226136537-bef5976b-7dbe-4a52-bfab-885a61862f57.png)
maybe mmdeploy can not support mmyolo

**Comment by kelvinwang139** [comment]
> Yes. It does. Please refer to https://github.com/open-mmlab/mmyolo/blob/main/docs/en/recommended_topics/deploy/mmdeploy_guide.md

I see yolov6 is supported in mmdeploy project，but I don't see the related information under mmdeploy related link:
https://mmdeploy.readthedocs.io/en/latest/04-supported-codebases/mmdet.html
